perspective_reddit_bot/moderate_subreddit.py

```diff
@@ -0,0 +1,152 @@
+"""A reddit bot which uses the Perpsective API to help moderate subreddits.
+
+Args:
+      subreddit: (required) A positional argument for the name of the subreddit to moderate.
+      -rule_config_file: (optional) The file which contains moderation rules. Defaults to 'rules.yaml'
+      -output_dir: (optional) If set, scores for all streamed commments are written to a file in this directory.
+
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
```

Comment:
Include the standard python 2/3 compatibility imports:

```
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```

Reply:
Done.